build/includes/sdk.mk

```diff
@@ -170,6 +170,17 @@ run-sdk-conformance-tests: run-sdk-conformance-test-node run-sdk-conformance-tes
 clean-sdk-conformance-tests:
 	$(MAKE) run-all-sdk-command COMMAND=clean
 
+# Start a shell in the SDK image. This is primarily used for publishing packages.
+# Usig a shell is the easiest, because of Google internal processes and interactive commands required.
+sdk-shell:
+	$(MAKE) ensure-build-sdk-image SDK_FOLDER=$(SDK_FOLDER)
+	docker run --rm -it $(common_mounts) -v ~/.ssh:/tmp/.ssh:ro $(DOCKER_RUN_ARGS) \
+ 	--entrypoint=/root/shell.sh $(SDK_IMAGE_TAG)
+
+# 1SDK shell for node
```

Comment:
Is the 1 a typo or some acronym?

Reply:
Typo, will fix! Thanks for catching that!